tests/parser/controllers/joints_motor_min_max_position/joints_motor_min_max_position.c

```diff
@@ -0,0 +1,140 @@
+#include <webots/motor.h>
+#include <webots/robot.h>
+#include <webots/supervisor.h>
+
+#include "../../../lib/ts_assertion.h"
+#include "../../../lib/ts_utils.h"
+
+#define TIME_STEP 32
+
+int main(int argc, char **argv) {
+  ts_setup(argv[0]);
+  ts_disable_output_log();
+  // This test suite verifies that if the position field, as set in the jointParameters, is outside of the bounds of the
+  // minPosition and maxPosition of RotationalMotor, these bounds must adadpt accordingly.
+  // This suite also checks that the correct pairing is done (device <-> jointParameters, device2 <-> jointParameters2)
+
+  // SliderJoint Tests -------------------------------------------------------------------------------------------------
+  WbDeviceTag slider_joint_motor_lower_limit = wb_robot_get_device("slider_joint_lower_limit_motor");
```

Comment:
It looks better here to match the variable name with the device name, isn't it?
```suggestion
  WbDeviceTag slider_joint_lower_limit_motor = wb_robot_get_device("slider_joint_lower_limit_motor");
```

Reply:
It does, thank you. Also I noticed the messages for the upper limit tests were swapped (correct test, wrong message) hence why there's more changes in  the last commit.

Reply:
Thank you. However, I am not very happy with the new names as I believe a device name should end with the device type, e.g., "my special distance sensor" or "left motor". Hence I would prefer its ends with "motor" in this case rather than having the "motor" word in the middle of the name, which looks odd to me.